src/CodeStyle/Core/Analyzers/PublicAPI.Unshipped.txt

```diff
@@ -1,3 +1,33 @@
+Microsoft.CodeAnalysis.Internal.Editing.DeclarationModifiers
```

Comment:
This is a public type defined in Workspaces, which is also needed for NamingStyle rules. It has been linked into the CodeStyle analyzers project under a different namespace `Microsoft.CodeAnalysis.Internal.Editing` versus `Microsoft.CodeAnalysis.Editing`

Reply:
does 'internal' mean we are not signed up to supporting this APi and can remove/break it at any time?

Reply:
CodeStyle packages are not really libraries, so there is no support. Sam added these files just to track dependencies from this layer.